File: python/prime.py
#!/usr/bin/python3

import math
import sys

min=int(sys.argv[1])
max=int(sys.argv[2])

def myrange(start,stop):
	i = start
	while i < stop:
		yield i
		i += 1

for i in myrange(min,max+1):
	prim=1
	for j in myrange(2,int(math.sqrt(i)+1)):
		if i % j == 0:
			prim=0
			break
	if prim == 1:
		print(i)

File: ruby/prime.rb
#!/usr/bin/ruby

min=ARGV[0].to_i
max=ARGV[1].to_i

for i in min..max
	prim=1
	for j in 2..Math.sqrt(i)
		if (i % j == 0) then
			prim=0
			break
		end
	end
	if (prim == 1) then
		puts i
	end
end

